docs/getting_started_guide/Installing_SIMP_From_A_Repository.rst

```diff
@@ -110,59 +110,85 @@ Rebuild The Yum Cache
 
    $ sudo yum makecache
 
-Install SIMP
--------------
-.. code-block:: bash
-
-   $ sudo yum install -y simp
+Install the SIMP Server
+-----------------------
 
-Modify Yum URLs
----------------
+1. Select the simp-adapter package appropriate for the version of Puppet
+   you will be using
 
-Set the following variables to repositories of your choosing in
-``/etc/puppetlabs/code/environments/production/hieradata/default.yaml``
+   * simp-adapter-foss:  Version appropriate for FOSS Puppet
+   * simp-adapter-pe:   Version appropriate for Puppet Enterprise
 
-.. code-block:: yaml
+2. Install the simp-adapter package
 
-   # Full URL to a YUM repo for Operating System packages
-   simp::yum::os_update_url: 'http://mirror.centos.org/centos/$releasever/os/$basearch/'
```

Comment:
simp config prompts for these URLs.  I'm not sure how you want to document this.